internal/service/streamprivatelinkendpoint/resource_test.go

```diff
@@ -1,34 +1,290 @@
+	dataSourcePluralName = "data.mongodbatlas_stream_privatelink_endpoints.test"
 )
 
-// TODO: if acceptance test will be run in an existing CI group of resources, the name should include the group in the prefix followed by the name of the resource e.i. TestAccStreamRSStreamInstance_basic
-// In addition, if acceptance test contains testing of both resource and data sources, the RS/DS can be omitted.
-func TestAccStreamPrivatelinkEndpointRS_basic(t *testing.T) {
-	// resource.ParallelTest(t, resource.TestCase{
-	// 	PreCheck:                 func() { acc.PreCheckBasic(t) },
-	// 	ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-	// 	//		CheckDestroy:             checkDestroyStreamPrivatelinkEndpoint,
-	// 	Steps: []resource.TestStep{ // TODO: verify updates and import in case of resources
-	// 		//			{
-	// 		//				Config: streamPrivatelinkEndpointConfig(),
-	// 		//				Check:  streamPrivatelinkEndpointAttributeChecks(),
-	// 		//			},
-	// 		//          {
-	// 		//				Config: streamPrivatelinkEndpointConfig(),
-	// 		//				Check:  streamPrivatelinkEndpointAttributeChecks(),
-	// 		//			},
-	// 		//			{
-	// 		//				Config:            streamPrivatelinkEndpointConfig(),
-	// 		//				ResourceName:      resourceName,
-	// 		//				ImportStateIdFunc: checkStreamPrivatelinkEndpointImportStateIDFunc,
-	// 		//				ImportState:       true,
-	// 		//				ImportStateVerify: true,
-	// 	},
-	// },
-	// )
+func TestAccStreamPrivatelinkEndpoint_basic(t *testing.T) {
+	tc := basicTestCase(t)
+	// Tests include testing of plural data source and so cannot be run in parallel
+	resource.Test(t, *tc)
+}
+
+func TestAccStreamPrivatelinkEndpoint_failedUpdate(t *testing.T) {
+	tc := failedUpdateTestCase(t)
+	// Tests include testing of plural data source and so cannot be run in parallel
+	resource.Test(t, *tc)
+}
+
+func basicTestCase(t *testing.T) *resource.TestCase {
+	t.Helper()
+
+	var (
+		// need specific projectID because feature is currently under a Feature flag
+		projectID           = os.Getenv("MONGODB_ATLAS_STREAM_AWS_PL_PROJECT_ID")
+		provider            = "AWS"
+		region              = "us-east-1"
+		awsAccountID        = os.Getenv("AWS_ACCOUNT_ID")
+		networkID           = "n-dz0vk2"
+		privatelinkAccessID = "pla-km26gn"
+	)
+
+	return &resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ExternalProviders:        acc.ExternalProvidersOnlyConfluent(),
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: getCompleteConfluentConfig(true, projectID, provider, region, vendor, awsAccountID, networkID, privatelinkAccessID),
+				Check:  checksStreamPrivatelinkEndpoint(projectID, provider, region, vendor, false),
+			},
+			{
+				Config:            getCompleteConfluentConfig(true, projectID, provider, region, vendor, awsAccountID, networkID, privatelinkAccessID),
```

Comment:
do you need the config in the import step?

Reply:
You are right, removed